components/net/bluetooth_thread.rs

```diff
@@ -35,7 +35,7 @@ const VALUE_ERROR: &'static str = "No characteristic or descriptor found with th
 const SECURITY_ERROR: &'static str = "The operation is insecure";
 const NETWORK_ERROR: &'static str = "A network error occurred";
 // A transaction not completed within 30 seconds shall time out. Such a transaction shall be considered to have failed.
-// https://www.bluetooth.org/DocMan/handlers/DownloadDoc.ashx?doc_id=286439 (p. 480)
+// https://www.bluetooth.org/DocMan/handlers/DownloadDoc.ashx?doc_id=286439 ([Vol 3, Part F] page 480)
```

Comment:
`(Vol. 3, page 480)`

Source: Modern Language Association of America's MLA Handbook for Writers of Research Papers, Seventh Edition.